pkgs/development/python-modules/i3ipc/default.nix

```
@@ -1,24 +1,33 @@
 { stdenv, buildPythonPackage, fetchFromGitHub
 , enum-compat
-, xorgserver, pytest, i3, python
+, xorgserver, pytest, pytest-xvfb, pytest-asyncio, i3, python, xlib, xdpyinfo
+, makeFontsConf, coreutils
 }:
 
 buildPythonPackage rec {
   pname = "i3ipc";
-  version = "1.6.0";
+  version = "2.0.1";
 
   src = fetchFromGitHub {
     owner  = "acrisci";
     repo   = "i3ipc-python";
     rev    = "v${version}";
-    sha256 = "0sb525wvwcnikjaqzha94xr97r1gjys30csmaj17swlxgyczxvq5";
+    sha256 = "10zpbiw1gcndn439g1vxcdkxllwp02qcmaal4w7hi2rzgaw1xkdk";
   };
+  propagatedBuildInputs = [ enum-compat pytest ];
+
+  fontsConf = makeFontsConf {
+    fontDirectories = [ ];
+  };
+  FONTCONFIG_FILE = fontsConf; # Fontconfig error: Cannot load default config file
+  checkInputs = [ coreutils pytest xdpyinfo pytest-asyncio pytest-xvfb xorgserver i3 xlib ];
```

coreutils is available during buildPhase
```suggestion
  checkInputs = [ pytest xdpyinfo pytest-asyncio pytest-xvfb xorgserver i3 xlib ];
```